moped-editor/src/views/projects/projectView/ActivityLogComponents/TagsActivityEntry.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Box, Typography, makeStyles } from "@material-ui/core";
+import LocalOfferOutlinedIcon from "@material-ui/icons/LocalOfferOutlined";
+
+const useStyles = makeStyles((theme) => ({
+  entryText: {
+    padding: "0 0 0 .5rem",
+  },
+  boldText: {
+    fontWeight: 700,
+  },
+}));
+
+const getEntryText = (change, tagList) => {
+  // Adding a new tag
+  if (change.description.length === 0) {
+    return `Project tagged with "${
+      tagList[change.record_data.data.new.tag_id]
+    }" `;
+  }
+  // Soft deleting a tag is the only update a user can do (is_deleted is set to true) can do
```

Comment:
Extra `can do` in your comment, and maybe it could read:

A user can only perform one type of update. They can soft-delete a tag which sets `is_deleted` to true.

Reply:
thanks :)